sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/query.sql

```diff
@@ -0,0 +1,84 @@
+SELECT
+  submission_date,
+  app_version,
+  os,
+  channel,
+  locale,
+  browser_version,
+  country,
+  newtab_homepage_category,
+  newtab_newtab_category,
+  organic_content_enabled,
+  sponsored_ads_enabled,
+  sponsored_topsites_enabled,
+  organic_topsites_enabled,
+  newtab_search_enabled,
+  SUM(all_visits) AS all_visits,
```

Comment:
I think all_visits should be a `COUNT(DISITNCT newtab_visit_id)`.

The idea is to also include in the count visits that do not have any engagement or do not fit the default_ui_visits. This would help compute the proportion of `default_ui_visits / all_visits` as the proportion of visits with default UI.

Reply:
all_visits is the count of distinct newtab_visit_id for all events identified [here](https://github.com/mozilla/bigquery-etl/blob/e056ef094ca7ed47a78d3116c52fef1c8cf8bdd7/sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_v2/query.sql#L30-L75) as defined in the [doc](https://docs.google.com/document/d/1UJpGhy6ZPW96eblZOd0WyP7N5VBVgN0aXi5T8GzqWU8/edit?tab=t.0#heading=h.i89uxxvt36rl).
The [filters](https://github.com/mozilla/bigquery-etl/blob/generated-sql/sql/moz-fx-data-shared-prod/telemetry_derived/newtab_visits_v1/query.sql#L583) in newtab_visits table does not seem identical to that definition. They seem to exclude: search.ad.click, sections and inline_selection_click

Reply:
> all_visits is the count of distinct newtab_visit_id for all events identified [here](https://github.com/mozilla/bigquery-etl/blob/e056ef094ca7ed47a78d3116c52fef1c8cf8bdd7/sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_v2/query.sql#L30-L75)

we will miss out on new tab visits that do not have any of those events fired, including non-default UI visits and visits that technically still have a default UI but for whatever reason don't see any of those events fired. 

This will 1) inflate the % of visits with a default UI over all the visits (including non-default UI) and 2) inflate engagement rates for default UI visits.

Reply:
synced with G and newtab open event also qualifies as a newtab visit